### PR TITLE
Revise desktop wallet docs

### PR DESCRIPTION
**Pull request by crypto-matto**
## Background
Revise Desktop Wallet documentations on: 
- Full DApp browser support
- Ledger UX update
- https://github.com/crypto-com/chain-desktop-wallet/pull/958

### CODE REVIEW

File: docs/wallets/desktop_wallet.md [hunk 1]
```diff
@@ -313,18 +313,30 @@ You can always review your latest bridge transfer history at the top-right corne
...
 ## 7. DApp Browser
-Since v0.6.1, you’re able to access various DApps directly (`Standard` Wallet Type) through the Desktop Wallet. Certain DApps are available at the moment, and we’re going to bring more supported DApps in the future. 
+Since v0.6.4, you’re able to access any DApps available on Cronos Mainnet Beta directly through the Desktop Wallet on both Standard & Ledger Wallet. 
         <img src="./assets/desktop_wallet/7.png" />
 
-### 7.1 - Connect to DApp
-To connect to a DApp, click the "connect" button and choose "**Metamask**". 
-        <img src="./assets/desktop_wallet/7-1-1.png" />
+### 7.1 - Choose your DApp
+You may go to any DApps you like by either clicking the DApps inside *Popular* / *Saved* section, or type the URL of the DApp directly on the Address Bar at the top.
```
Comment (aw126) [inline docs/wallets/desktop_wallet.md:325]:
```suggestion
You may go to any DApps you like by either clicking the DApps inside the *Popular* / *Saved* section or typing the URL of the DApp directly on the Address Bar at the top. 
```

File: docs/wallets/desktop_wallet.md [hunk 2]
```diff
@@ -313,18 +313,30 @@ You can always review your latest bridge transfer history at the top-right corne
...
         <img src="./assets/desktop_wallet/7.png" />
 
-### 7.1 - Connect to DApp
-To connect to a DApp, click the "connect" button and choose "**Metamask**". 
-        <img src="./assets/desktop_wallet/7-1-1.png" />
+### 7.1 - Choose your DApp
+You may go to any DApps you like by either clicking the DApps inside *Popular* / *Saved* section, or type the URL of the DApp directly on the Address Bar at the top. 
+
+### 7.2 - Save your favourite DApp
+You can always save any DApps you like by clicking the bookmark button at the top-right corner. When the bookmark button turns blue, the DApp has been successfully saved into your *Saved* collection.
```
Comment (aw126) [inline docs/wallets/desktop_wallet.md:328]:
```suggestion
You can easily save any DApps you like by clicking the bookmark button at the top-right corner. When the bookmark button turns blue, it means the DApp has been successfully saved into your *Saved* collection.
```

File: docs/wallets/ledger_desktop_wallet.md [hunk 3]
```diff
@@ -104,8 +104,14 @@ Desktop Wallet supports all sorts of transactions for both normal wallets and wa
 Fill up the request information: recipient address, amount to be sent and optional memo as needed:
             <img src="./assets/ledger_desktop_wallet/s4-1-2.png" />
 
-- **Step 2** - Once you hit the "*Continue*" button, you can review the transaction details in the confirmation window on the desktop wallet and proceed to the next step by clicking "*Confirm*":
-            <img src="./assets/ledger_desktop_wallet/s4-2.png" />
+- **Step 2** - Once you hit the "*Continue*" button, you can review the transaction details in the confirmation window on the desktop wallet. Proceed to the next step by clicking "*Confirm*":
+            <img src="./assets/ledger_desktop_wallet/s4-2-1.png" />
+
+**Remark**: If you have not yet established a connection with your Ledger device, a notification will popup on the top-right hand corner, asking you to confirm connection with your Ledger.
```
Comment (aw126) [inline docs/wallets/ledger_desktop_wallet.md:110]:
```suggestion
**Remark**: If you have not yet established a connection with your Ledger device, a notification will pop up on the top-right hand corner, asking you to confirm the connection with your Ledger.
```

File: docs/wallets/desktop_wallet.md [hunk 4]
```diff
@@ -316,6 +316,11 @@ If you found your bridge transfer history failed to get latest transferrals, ple
 Since v0.6.4, you’re able to access any DApps available on Cronos Mainnet Beta directly through the Desktop Wallet on both Standard & Ledger Wallet. 
         <img src="./assets/desktop_wallet/7.png" />
 
+::: warning DISCLAIMER
+You are responsible for your use of any DApps. DApps are controlled solely by their respective project providers. Please check the receiving addresses or contract addresses before you transfer any assets.
+We are not responsible for the accuracy, completeness, or usefulness of such DApps. Accordingly, we neither endorse, recommend, nor give any opinion, advice or whatsoever on such DApps. are not responsible or liable for any losses incurred during your use of DApps.
```
Comment (aw126) [inline docs/wallets/desktop_wallet.md:321]:
```suggestion
We are not responsible for the accuracy, completeness, or usefulness of such DApps. Accordingly, we neither endorse, recommend, nor give any opinion, advice or whatsoever on such DApps, and are not responsible or liable for any losses incurred during your use of DApps.
```

**Comment by aw126** [comment]
Looks good - just made a few minor changes.

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**